src/sched/lthread.c

```diff
@@ -1079,3 +1079,23 @@ void lthread_set_expired(struct lthread* lt)
 {
     lt->attr.state |= BIT(LT_ST_EXPIRED);
 }
+
+#ifdef DEBUG
+void lthread_dump_all_threads(void)
+{
+    struct lthread_queue* lt_queue = __active_lthreads;
+
+    SGXLKL_VERBOSE("Dumping all lthreads:\n");
+
+    for (int i = 1; lt_queue; i++)
```

Comment:
The `1` here is probably correct, but I'm missing the context to say so for sure.

Reply:
We use this to prefix the printed lines, and this can start from 1 when counting the threads.